test/saucelabs/karma.conf.js

```diff
@@ -66,13 +66,20 @@ module.exports = config => {
         watched: true,
         served: true
       },
+      {
+        pattern: "test/specs/*.spec.mjs",
+        included: true,
+        watched: true,
+        served: true,
+        type: "module"
+      },
       {
         pattern: "test/**/*.+(svg|png|jpg|jpeg|ttf|txt)",
         included: false,
         served: true
       },
       {
-        pattern: "test/reference/**/*.pdf",
+        pattern: "test/reference/**/*.*",
```

Comment:
don't think that's necessary. The *.ttf files should be served correctly by the previous rule.

Reply:
This was more of a test to see if there was something weird going on with serving the fonts on saucelabs. I just copied the pattern from `unit/karma.conf.js` since it was working locally.

I'll revert it, but would you prefer that I do a `git revert` or `git reset HEAD^ --hard`?

Reply:
Since it was just a test, I'll reset the commit. No point in having two unnecessary commits.

Reply:
I'll squash the PR commits anyway when merging, so it won't make a difference in the end.